spec/domain/audits/find_content_spec.rb

```diff
@@ -45,6 +46,20 @@ module Audits
       end
     end
 
+    describe '#my_content' do
+      subject(:relation) { described_class.my_content(user.uid) }
+
+      before do
+        Audits::Allocation.create(uid: user.uid, content_id: Content::Item.first.content_id)
```

Comment:
We should probably use a factory for these - it's in keeping with the rest of the tests, and easier to maintain (eg if we were to add a mandatory field to `Allocation`).

Reply:
👌